.eslintrc.json

```
@@ -0,0 +1,17 @@
+{
+  "plugins": [
+    "@typescript-eslint"
+    
+  ],
+  "extends": [
+    "airbnb-base", 
+    "airbnb-typescript/base",
+    "plugin:@typescript-eslint/recommended",
+    "prettier"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": "latest",
+  },
+  "rules": {}
+}
```

```suggestion
{
  "plugins": [
    "@typescript-eslint", // mandatory typescript plugin
    "sonarjs" // sonar plugin for code analysis
  ],
  "extends": [
    "airbnb-base", // use instead on eslint recommended as a stricter alternative
    "airbnb-typescript/base", // airbnb rules overrides for typescript
    "plugin:@typescript-eslint/recommended",
    "plugin:sonarjs/recommended", // sonar
    "prettier" // compatibility with prettier (disable conflicting rules) NB: this is the `eslint-config-prettier` it should not be duplicated
  ],
  "parser": "@typescript-eslint/parser",
  "parserOptions": {
    "ecmaVersion": "latest",
    "project": "./tsconfig.json" // Specify the path to your tsconfig.json file
  },
  "rules": {}
}
```
use .eslint propose by hardhat. Moreover the last config didn't work on my side